docs/integrate/queryzen/index.md

```diff
@@ -0,0 +1,72 @@
+(queryzen)=
+# QueryZen
+
+:::{rubric} About
+:::
+
+[QueryZen] makes it easier to manage SQL query statements over HTTP.
+
+A `Zen` in QueryZen jargon is a named, parameterized and versioned SQL query
+that is created, updated and executed over HTTP endpoints. It allows you to
+decouple SQL from your application while controlling, versioning, and securing
+data access from development to production.
+
+![](https://github.com/surister/queryzen/raw/master/queryzen-docs/docs/concepts/img.png){w=700px}
+
+:::{rubric} Features
+:::
+
+Overview:
+
+* Create, get and delete Zens in different collections and run them in different Databases.
+* Automatically version queries, name and safely parameterize queries with special functions.
+* High level of Coverage and Tests.
+* Track, save and analyze statistics of your queries over time and versions.
+* Everything is dockerized for easy development and deployment.
+
+QueryZen includes:
+
+* HTTP REST backend to handle the lifetime of Zens.
+* Task execution backend to handle the execution of the queries.
+* Database driver abstraction for Python SQL drivers.
+* Pythonic package to programmatically use QueryZen.
+
+With QueryZen you can:
+
+* Quickly create HTTP Rest endpoints of your SQL data.
+* Integrate your SQL data in your data pipelines with minimal configuration.
+* Monitor individual query executions and analyze metrics.
+* Version your SQL queries, build and test queries without affecting production.
+* Create materialized views for SQL databases that do not support them.
+
```

Comment:
Maybe add a rubric (subsection) about "Install" matters, with concise incantation examples for both the server via `docker` or `podman` and the client via `uv`?